package.xml

```diff
@@ -13,10 +13,12 @@
   <buildtool_depend>rosidl_parser</buildtool_depend>
 
   <build_depend>pkg-config</build_depend>
+  <build_depend>python3-yaml</build_depend>
   <build_depend>rclcpp</build_depend>
   <build_depend>rmw_implementation_cmake</build_depend>
   <build_depend>std_msgs</build_depend>
 
+  <exec_depend>python3-yaml</exec_depend>
```

Comment:
Since the package does install the Python code it is good to declare this. I just assume nobody is using that without actually rebuilding the bridge :wink: